### PR TITLE
lxqt-sudo: lxsu & lxsudo symlinks for lxqt-sudo

### PR DESCRIPTION
**Pull request by palinek**
closes lxde/lxqt#794

### CODE REVIEW

**Comment by jleclanche** [comment]
Why `-f`?

**Comment by palinek** [comment]
> Why -f?

The symlink could exist from previous build...

File: sudo.cpp [hunk 1]
```diff
@@ -1,116 +1,269 @@
...
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
-
- * You should have received a copy of the GNU Lesser General
- * Public License along with this library; if not, write to the
- * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
- * Boston, MA 02110-1301 USA
- *
- * END_COMMON_COPYRIGHT_HEADER */
```
Comment (paulolieuthier) [inline sudo.cpp]: No license?

Comment (palinek) [inline sudo.cpp]: That's a mistake, of course.

**Comment by paulolieuthier** [comment]
@palinek GTM. Please squash/rename the commits as you with and ship it. @pmattern I think a issue must be files in pcmanfm-qt's bugtracker.

**Comment by palinek** [comment]
@paulolieuthier actually there is a discussion in lxde/lxqt#794 about the naming etc.... and I'm confused if the lxsudo/lxsu is wanted.

**Comment by palinek** [comment]
@pmattern with @3471832 lxqt-sudo supports multiple arguments for su backend... so no need for quotting in pcmanfm-qt configuration.

**Comment by paulolieuthier** [comment]
> Are you sure about this? I moved the argument size check after "options removal" intentionally. It's a check if a command to run was provided.

Sorry, I was just trying to fix a segfault.

> @paulolieuthier actually there is a discussion in lxde/lxqt#794 about the naming etc.... and I'm confused if the lxsudo/lxsu is wanted.

Yeah, I'm trying to end that discussion. Apparently, it has come to some agreement.

**Comment by palinek** [comment]
> > Are you sure about this? I moved the argument size check after "options removal" intentionally. It's a check if a command to run was provided.

> Sorry, I was just trying to fix a segfault.

I see it now.... updated.

**Pull request merged**